tensorboard/webapp/widgets/data_table/filter_dialog_component.scss

```diff
@@ -30,15 +30,49 @@ limitations under the License.
       'background'
     );
   }
+}
+
+.input-container {
+  margin-bottom: 8px;
+}
+
+.discrete-filters-container {
+  max-height: 300px;
+  overflow-y: auto;
+}
+
+.no-matches {
+  // 12px is the width of the checkbox so the text should match the
+  // indentation of the selectable filters.
+  padding: 8px 12px;
+}
 
-  .discrete-filters-area {
-    max-height: 100px;
-    overflow-y: auto;
+.filter-container {
+  padding-bottom: 8px;
```

Comment:
Would it be preferable to remove the padding so that there is no gap between the scrollbar and the border when you have many discrete items? It looks slightly off when you have enough items for a scrollbar.

Unfortunately my suggestion comes at the cost of things being more cramped when:
* you have few-enough discrete items that there is no scrollbar
* you are using the interval filter (although in this case you could apply an additional class to get the padding)

Reply:
I'm not certain that I understand this suggestion. Are you suggesting using asymmetrical padding?

We could just JS to add another class in the event there are enough items to cause it to scroll, but that seems like a bit of an arms race. Would it satisfy your concerns about cramping?

Reply:
This could be an interesting thing to do, but since we're launching next week I'd rather get this cleanup in and potentially improve the styling further in the future.